bagel/tests/data/README.md

```diff
@@ -2,24 +2,25 @@
+| 4            | valid, has additional columns not described in `.json`                          | same as example 1                                                                    | pass               |
+| 5            | valid, has additional unique value, not documented in `.json`                   | same as example 1                                                                    | fail               |
+| 6            | valid, same as example 5. has annotation tool columns                           | valid, contains `"MissingValues"` attribute for categorical variable                 | pass               |
+| invalid      | valid, only exists to be used together with the (invalid) .json                 | invalid, missing the `"TermURL"` attribute for identifiers                           | fail               |
+| 7            | has fewer columns than are annotated in `.json`                                 | same as example 1                                                                    | fail               |
+| 8            | valid, based on ex2 has multiple participant_id columns                         | valid, based on ex2 multiple participant_id column annotations                       | fail*              |
+| 9            | invalid, based on example 6 but contains an unannotated value for `group`       | valid, based on example 6                                                            | fail               |
+| 10           | valid, same as example 6                                                        | valid, based on example 6 but contains extra `"MissingValues"` not found in the .tsv | pass, with warning |
+| 11           | invalid, ex 6 with missing entries in `participant_id` and `session_id` columns | valid, based on example 6                                                            | fail               |
+| 12           | Valid, same as example 2                                                        | Valid, based on example 2 but missing BIDS "Levels" attribute for group column       | Pass, with warning |
+| 13           | Valid, same as example_synthetic                                                | Valid, based on example_synthetic but with mismatched levels for group column        | Pass, with warning |
+| 14           | Valid, same as example 2                                                        | Valid, based on example 2, but with an extra column annotation without Neurobagel    | Pass               |
+| 15           | Valid, same as example 2                                                        | Invalid, based on example 2, but participant ID column lacks Neurobagel annotations  | Fail               |
+| 16           | Invalid, same as example2.csv, but with a sneaky .tsv file ending               | Valid, same as example2                                                              | fail               |
```

Comment:
Just a thought -

With larger README markdown tables like this I think we're starting to see the downsides of using an auto-table formatter in our code editor. I realized the reason for the messy diffs is because generally the entire table is adjusted when a new row is added (assuming it has contents of a different length than the existing rows).

I think in future, except for very small tables it may be better to just turn off auto-formatting and stick to a basic markdown table format, e.g.

```markdown
column 1 | column 2 | column 3
---- | ---- | ----
my | first | row
my | second longer | row
my | third | row
```
This will restrict diffs to just the modified/added rows. It won't look as neat in markdown, but even with auto-formatting, as the table gets bigger, the reformatted table still ends up with messy text wrapping except on a very wide screen anyways.

Reply:
Agree. These examples are also getting a bit long.